packages/r2frida/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/nowsecure/r2frida
+TERMUX_PKG_DESCRIPTION="Frida IO plugin for radare2"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="5.8.6"
+TERMUX_PKG_SRCURL=https://github.com/nowsecure/r2frida/archive/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=c65979581dc514e58639b5c9492ed21780370446fad6d8cfcaaffe25e86366f1
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS="radare2"
+TERMUX_PKG_BUILD_DEPENDS="radare2"
+
+termux_step_pre_configure () {
+	if [ $TERMUX_ARCH != "aarch64" ]; then
+		termux_error_exit "Unsupported arch: $TERMUX_ARCH"
+	fi
+	unset CXXFLAGS
+	unset CPPFLAGS
+	unset LDFLAGS
+	unset CFLAGS
```

Comment:
This will cause issues, compilation will not target android (unless the Makefiles sets everything up)

Reply:
Builds fine for me if i spawn a shell right there. The problem is that it cant execute r2

Reply:
Alright, maybe they aren't used, is it required to unset them?